roles/openshift_manage_node/tasks/node_ready.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Wait for node to be ready
+  oc_obj:
+    state: list
+    kind: node
+    name: "{{ openshift.node.nodename | lower }}"
+  register: node_output
+  until: node_output.results.returncode == 0 and node_output.results.results[0].status.conditions | selectattr('type', 'match', '^Ready$') | map(attribute='status') | join | bool == True
+  # Give the node two minutes to come back online.
+  retries: 24
+  delay: 5
+  delegate_to: "{{ openshift_master_host }}"
```

Comment:
We may have situations where the nodes never go ready because they require external CSR signing. Lets tolerate failure here and move on in those cases.